erpnext_documentation/www/docs/v13/user/manual/en/manufacturing/production-plan.md

```diff
@@ -43,6 +43,9 @@ A Production Plan can also be created manually where you can select the Items to
 
 1. On expanding a row in the Items to Manufacture table, you'll see an option to 'Include Exploded Items'. Ticking this includes raw materials of the sub-assembly items in the production process.
 
+1. If "Consolidate Items" is ticked and saved, items with the same BOM are combined into a single item with the combined total planned quantity.
+![Get items Combined for Production Plan]({{docs_base_url}}/v13/assets/img/manufacturing/get_items_combined_wo.png)
```

Comment:
```suggestion
![Get items combined for Production Plan](/docs/v13/assets/img/manufacturing/get_items_combined_wo.png)
```

Not sure why `{{docs_base_url}}` is used in this file. Maybe this is the legacy way? An [absolute path-reference](https://datatracker.ietf.org/doc/html/rfc3986#section-4.2) should work well enough, without the need of jinja variables.